### DISCUSSION TITLE
[newbie] Can I disable "normal web" in Beaker?

### DISCUSSION THREAD

**Discussion by akavel** [beakerbrowser/beaker#1684] [General]
I haven't even downloaded BB yet, but before doing that I'd like to ask: can I disable "normal web" in it? I normally use Firefox for browsing regular web, I would prefer to use Beaker only for the distributed web.

**Comment by pfrazee** [comment]
You can choose not to make beaker the default OS handler for "http/s" links. Are you looking for more than that?

**Reply by akavel** [reply]
Yes, more:

- if *a hyper:// website* tries to download something from http/s (e.g. a tracking pixel, or some tracking JS, or actually any image or JS or iframe), I would like this request to be blocked
- if I click/paste a non-hyper URL (i.e. a http/s one) into Beaker, I'd like Beaker to refuse downloading it & probably display some info like "http(s) is blocked"; the "blocked" screen could also have a button "[Open the page in your system browser]", which when clicked would pass the URL to my system Firefox

**Reply by pfrazee** [reply]
We could certainly look into a "lockdown mode" like this, but we don't have that yet

**Reply by akavel** [reply]
I kinda started to think, maybe this could be somehow done already with some kinda smart proxy script? I think I saw once that proxy settings in browsers can actually be full-blown JS scripts? Though I'm not 100% sure about this...
